pytest_sugar.py

```diff
@@ -625,3 +624,8 @@ def print_failure(self, report):
                 self.write_sep("―", msg)
                 self._outrep_summary(report)
         self.reset_tracked_lines()
+
+
+# On older version of Pytest, allow default progress
+if parse(pytest.__version__) <= parse('3.4'):  # pragma: no cover
```

Comment:
Just out of curiosity: why the "# pragma: no cover" here?
Shouldn't it either get tested in some build job, or otherwise show up as not being covered then?

Reply:
It is covered on some build job but only old version are supposed to cover this.
Do you know if codecov will accumulate all the build coverage?

Reply:
Yes.. it also uses flags for different builds.

Reply:
Will address this when rebasing https://github.com/Frozenball/pytest-sugar/pull/148.

Reply:
Thanks for the PR!
I hope we get it released soon.
/cc @Frozenball